install/_lib.sh

```diff
@@ -45,7 +48,7 @@ fi
 # A couple of the config files are referenced from other subscripts, so they
 # get vars, while multiple subscripts call ensure_file_from_example.
 function ensure_file_from_example {
-  if [[ -f "$1" ]]; then
+  if [[ -f $1 ]]; then
```

Comment:
actually a little worried about pre-commits modifying this, could be a problem if $1 is a string with spaces in it. I noticed this at least when sending envelopes

Reply:
Sometimes we actually want to wrap that in quotes

Reply:
https://unix.stackexchange.com/questions/86800/difference-between-quoting-variables-in-shell-script-if-statements

Apparently if $1 is null we'll have an issue if it's not in quotes too

Reply:
This was due to passing `--simplify`, which I have now removed. Let me know if you notice anything else amiss.